WordPress/Classes/ViewRelated/System/Sidebar/SidebarViewController.swift

```diff
@@ -126,10 +127,17 @@ private struct SidebarView: View {
                 Label(Strings.domains, systemImage: "network")
             }
         }
-#endif
+        // A dedicated profile view is displayed at the bottom when user is signed in.
+        if viewModel.account == nil {
+            Button(action: { viewModel.navigate(.profile) }) {
```

Comment:
When you are not signed-in, let's change the profile view at the bottom to the following:

<img width="440" alt="Screenshot 2024-08-28 at 3 10 52 PM" src="https://github.com/user-attachments/assets/74f55ebf-b2cd-440b-a255-69d0387a16e9">

```swift
              Button(action: {}) {
                        HStack {
                            Image(systemName: "person.crop.circle")
                                .font(.title2)
                            VStack(alignment: .leading, spacing: 1) {
                                Text("Sign In")
                                Text("WordPress.com")
                                    .font(.footnote)
                                    .foregroundColor(.secondary)
                            }
                        }
                    }
```

The gearshape would still open the "Me" view with the same content as before. Let's remove its opacity and slightly bump the size. 

```swift
          Image(systemName: "gearshape")
                .font(.title3)
                .foregroundColor(Color.secondary)
```

Reply:
I think it works without the footnote too, but I'd rather clear about it.

<img width="470" alt="Screenshot 2024-08-28 at 3 14 14 PM" src="https://github.com/user-attachments/assets/5f8f1c44-7d25-4d12-b799-a40edc67c009">

Reply:
@jkmassel your feedback would also be appreciated. TL;DR: with the "Me" tab gone, we need a good replacement for it for the scenario when you are not logged in.

Reply:
The Jetpack app will pretty much force you to sign in with WP.com, so this is mostly a WordPress app problem.

In that context, it seems to me that it'd cause more confusion than benefit.

Could we just have it say "⚙️ Settings" in the secondary grey? The green is a pretty primary action

Reply:
This change is for a rare scenario when you have a self-hosted site but are logged-out in jpios.

I haven't yet considered how wpios will look like considering that it doesn't even have a tab bar now. I agree that there is less need to prioritize wp.com login in it. Since there are no tabs in wpios, we could leverage that sidebar by showing the item from "Me" directly in it: "App Settings", "About", etc.

Reply:
The login flow is also currrntly the same in wpios 🤔 It does prioritize wp.com accounts.

Reply:
I have taken most of you all's suggestions. What do you think about this?

|   | Jetpack | WordPress |
| - | ------- | --------- |
| without WP.com account | ![jp-without-account](https://github.com/user-attachments/assets/d41ed5aa-4330-47cf-87b1-d9194fb99f04) | ![wp-without-account](https://github.com/user-attachments/assets/c6510704-242a-48bc-b1c2-dfb23b6bd0a9) |
| with WP.com account | ![jp-with-account](https://github.com/user-attachments/assets/431b99e8-b391-4c96-8c6e-768c7246d5ed) | ![wp-with-account](https://github.com/user-attachments/assets/1cb8b8fb-dd23-4815-8c78-9cedc727f1f3) |

> Since there are no tabs in wpios, we could leverage that sidebar by showing the item from "Me" directly in it: "App Settings", "About", etc.

I didn't make this change, because I worry that would overcrowd the sidebar. There are many different things in the "Me" screen.

Reply:
Works great!

I made a couple of small tweaks:

- Reverted the profile color to label/secondaryLabel
- Add the "Gear" in the WordPress app to allow you accessing App Settings and other stuff, including the "Log in" button
- Replaced `var content: some AnyView?` with `@ViewBuilder var content: some View` (Apple doesn't recommend using `AnyView`, but it's unclear exactly how it affects the system)

The selection style in the sidebar seems odd, especially on iOS 18. I added it to the list of defects. I currently maintain this privately, but I can create a board or something like that.